resources/views/sale_pos/receipts/slim2.blade.php

```diff
@@ -445,7 +455,11 @@
                 	<strong>{!! $receipt_details->subtotal_label !!}</strong>
                 </p>
                 <p class="width-50 text-right">
-                	<strong>KSh {{(float)preg_replace('/[^0-9.]/', '', $receipt_details->subtotal) - (float)preg_replace('/[^0-9.]/', '', $receipt_details->taxes[array_key_last($receipt_details->taxes)])}}</strong>
+                	{{-- <strong>KSh {{(float)preg_replace('/[^0-9.]/', '', $receipt_details->subtotal) - (float)preg_replace('/[^0-9.]/', '', $receipt_details->taxes[array_key_last($receipt_details->taxes)])}}</strong> --}}
+
+                	<strong>KSh {{number_format((float)preg_replace('/[^0-9.]/', '', $receipt_details->total) - (float)$totalTax, 2)}}</strong>
```

Comment:
please create a helper function for preg_replace() and use that function calling with arguments.

Reply:
$sell_line->product->tax == 1 is to determine wheather this product accept tax or it is zero tax product. 1= tax applicable, 2= zero tax

Reply:
this is how preg_replace() is used in the whole invoice system, refactoring one will not help. If want to refactor preg_replace(), then need to refactor the whole invoice system

Reply:
done. check now